bolt-design-system/bolt
# Feature/micro journey images assets

## Asana

https://app.asana.com/0/1126340469288208/1137561697872881/f
https://app.asana.com/0/1126340469288208/1133408741134194/f
https://app.asana.com/0/1126340469288208/1136261310247059/f

## Summary

Refactors Connection to use the animated connection band. Only one option at the moment as we're waiting for the triple band animated svg. Svg can be flipped.
Refactors Character to use images local to the micro journey package, or a custom URL. 
Images are now bundled into the micro-journey package.

## How to test

Using the editor, you should be able to select a character and picks its image, or select "custom" and pass in an image url.
Using the editor, you can select the connection band. There is only one connection image to pick from at the moment, but you can flip its direction easily.
